src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -41,11 +41,12 @@
     <DocumentationFile>
     </DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)'!='Unix'">
+    <EnableNode>True</EnableNode>
+    <EnablePowerShell>True</EnablePowerShell>
+    <DefineConstants>$(DefineConstants);WINDOWS;FEATURE_NODE;FEATURE_POWERSHELL</DefineConstants>
```

Comment:
I did the various switched to keep the semantics of the individual changes clearer and make it easier to add node/powershell support on unix in future.

i.e.

* `MONO`: changes needed to run on the mono _runtime_ (which can run on windows).
* `WINDOWS`: things that will only work on the windows _platform_.
* `FEATURE_NODE`: the Node feature, which is not yet implemented for Unix (but could be)
* etc.

I imagine these annotations might be useful for the .NET Core port?